src/aiida/engine/launch.py

```diff
@@ -111,8 +111,14 @@ def submit(
         raise InvalidOperation('Cannot use top-level `submit` from within another process, use `self.submit` instead')
 
     runner = manager.get_manager().get_runner()
+
+    if runner.controller is None:
+        raise InvalidOperation(
+            'Cannot submit because the runner does not have a process controller. Does the current profile define a '
+            'broker? Submitting processes requires a broker like RabbitMQ.'
+        )
```

Comment:
Thanks @sphuber 
In addition, I think it would be useful to provide a solution, or clarify how else they could proceed without installing RabbitMQ. 
"You cannot submit your workflow because RabbitMQ is not installed, without RabbitMQ you can only run workflows"

Reply:
Good point. I will add that they should use `aiida.engine.launch.run` instead of `aiida.engine.launch.submit`.

Reply:
Done